packages/clerk-js/src/core/tokenCache.ts

```diff
@@ -63,7 +65,9 @@ const MemoryTokenCache = (prefix = KEY_PREFIX): TokenCache => {
     });
 
     const key = cacheKey.toKey();
-    const value: TokenCacheValue = { entry };
+
+    const createdAt = Math.floor(Date.now() / 1000);
+    const value: TokenCacheValue = { entry, createdAt };
 
     const deleteKey = () => {
       if (cache.get(key) === value) {
```

Comment:
🙃 i'm not a big fun of this check since it compares 2 `TokenCacheValue` objects and this equality operator is checking them not by value but by reference! I think we could something better here.

Reply:
Hey Dimitri, I might be wrong but I think this is meant to compare by reference, actually, but it's possible that we no longer need it. 

Keep in mind that before the setActive PR that we merged yesterday, the keys did not include the updateAt value. The `set` method schedules a delete call `expiresIn` seconds after caching the entry, while `get` might actually delete and cache an entry `leeway` seconds before the delete timeout is called.

The equality check in essence means that calling `deleteKey` will only delete the entry that scheduled the call and only that, even if a new entry with the same key was stored afterward.

LMK if this makes sense :) 

However, now that we do have the updatedAt in the key, let's think about whether we need to keep the check or not. I think i'd keep it, as the key is produced outside of that class so no one enforces that all entries will have different keys. Ofc, we can always refactor this logic and remove the check. LMK your thoughts